src/main/java/Client/MopeLSPClient.java

```diff
@@ -98,6 +98,17 @@ public String hover() throws ExecutionException, InterruptedException {
         return hoverGet.toString();
     }
 
+    public String getDocumentation(String className){
+        try{
+            CompletableFuture<String> x = server.getModelicaService().getDocumentation(className);
+            return x.get();
+        }catch(Exception e){
+            logger.error("Error CheckModel",e);
```

Comment:
The "CheckModel" looks like a copy-paste error.

Reply:
Upsi:)

Reply:
Fixed in https://github.com/MopeSWTP-SS21/MopeSWTP/commit/bcdb671991695389ca1d1d001f1350dc2a40bb12